sdk/tables/azure-data-tables/samples/async_samples/sample_custom_encoder_dataclass_async.py

```diff
@@ -0,0 +1,199 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_custom_encoder_dataclass_async.py
+
+DESCRIPTION:
+    These samples demonstrate the following: inserting entities into a table
+    and deleting tables from a table.
```

Comment:
Nit: Did you mean deleting entities from a table? Can also maybe mention that this sample also updates entities in a table as well.

Reply:
Same, it should be "deleting entities".